lib/shared/utils.js

```diff
@@ -78,3 +78,28 @@ exports.percentageToFloat = (percentage) => {
 
   return percentage / exports.PERCENTAGE_MAXIMUM
 }
+
+/**
+ * @summary Convert milliseconds to seconds
+ * @function
+ * @public
+ *
+ * @param {Number} time - millisecond time
+ * @returns {Number} second time
+ *
+ * @example
+ * const DELAY = 10000
+ * setTimeout(() => {
+ *   console.log(`${utils.millisecondsToSeconds(DELAY)} seconds passed.`)
+ * }, DELAY)
+ */
+exports.millisecondsToSeconds = (time) => {
+  if (!_.isNumber(time)) {
+    throw errors.createError({
+      title: `Not a number: ${time}`
+    })
+  }
+
+  // eslint-disable-next-line no-magic-numbers
+  return time / 1000
```

Comment:
Probably no harm in adding a `MILLISECONDS_IN_A_SECOND` constant somewhere?

I just did a search, and found `MILLISECONDS_IN_A_DAY` (and `daysToMilliseconds`) in `shared/units.js` so this whole function should probably be moved there?